stable/atlantis/README.md

```diff
@@ -35,6 +35,7 @@ The following options are supported.  See [values.yaml](values.yaml) for more de
 | `environment`                               | Map of environment variables for the container.                                                                                                                                                                                                                                                           | `{}`    |
 | `imagePullSecrets`                          | List of secrets for pulling images from private registries.                                                                                                                                                                                                                                               | `[]`    |
 | `gitconfig`                                 | Contents of a file to be mounted to `~/.gitconfig`.  Use to allow redirection for Terraform modules in private git repositories.                                                                                                                                                                          | n/a     |
+| `command`                                   | You can override basic container command parameter                                                                                                                                                                                                                                                        | n/a     |
```

Comment:
```suggestion
| `command`                                   | Optionally override the arguments passed to the Atlantis Docker image. By default no arguments are passed which is the same as passing `["server"]. Must be an array.`                                                                                                                                                                                                                                                        | n/a     |
```

Reply:
done

Reply:
I have really bad English. And your comment is better, but I think it's can be a different meaning. Please check that https://github.com/kubernetes/kubernetes/issues/8235 In any case, this is enough for me. Thank you for your feedback.

Reply:
Yeah you're right.. What do you need for your use-case? To override command or args?

Reply:
I need override command. Maybe you write another description and I add comment? My case will be fine with this code. Is description ok for you?